src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java

```diff
@@ -83,6 +83,12 @@ public enum Mixins {
     TRANSPARENT_CHAT(new Builder("Transparent Chat").setPhase(Phase.EARLY)
             .addMixinClasses("minecraft.MixinGuiNewChat_TransparentChat").setSide(Side.CLIENT)
             .setApplyIf(() -> Common.config.transparentChat).addTargetedMod(TargetedMod.VANILLA)),
+    LONGER_MESSAGES_CLIENT(new Builder("Longer Messages").setPhase(Phase.EARLY)
+            .addMixinClasses("minecraft.MixinGuiChat_LongerMessages").setApplyIf(() -> Common.config.longerSentMessages)
+            .addTargetedMod(TargetedMod.VANILLA).setSide(Side.CLIENT)),
+    LONGER_MESSAGES_SERVER(new Builder("Longer Messages Server Side").setPhase(Phase.EARLY)
+            .addMixinClasses("minecraft.MixinC01PacketChatMessage_LongerMessages").setApplyIf(() -> true)
```

Comment:
I think the reason we put config options everywhere is so we can have maximum compatibility with other mods and disable features if they cause crashes for certain installations. The proper way to code that feature would be to make the server send a packet to the client informing it of the chat message length authorized on that server. But ngl I don't know how to do that.

Reply:
Same. This is what I could come up with.